### PR TITLE
[GCP-8479] Bigtable: 'test_bigtable_list_tables' snippet flakes with '504 Deadline Exceeded'.

### PR DESCRIPTION
**Pull request by sangramql**
Addresses [#8479](https://github.com/googleapis/google-cloud-python/issues/8479)

Preliminary discussion can be found [here](https://github.com/sangramql/google-cloud-python/pull/6#issuecomment-516016294)

### CODE REVIEW

**Comment by sangramql** [comment]
@mf2199 please review,

**Comment by sangramql** [comment]
@mf2199 please review

File: bigtable/docs/snippets.py [hunk 1]
```diff
@@ -413,14 +417,6 @@ def test_bigtable_create_table():
...
-    from google.cloud.bigtable import column_family
-
-    client = Client(admin=True)
-    instance = client.instance(INSTANCE_ID)
-    table = instance.table("to_list")
-    max_versions_rule = column_family.MaxVersionsGCRule(2)
-    table.create(column_families={"cf1": max_versions_rule})
 
     # [START bigtable_list_tables]
     from google.cloud.bigtable import Client
```
Comment (mf2199) [inline bigtable/docs/snippets.py]: Yes, not sure why this duplicate import was there before, in the first place.

File: bigtable/docs/snippets.py [hunk 2]
```diff
@@ -471,9 +461,10 @@ def test_bigtable_delete_instance():
 
     client = Client(admin=True)
 
-    instance = client.instance("inst-my-123", instance_type=PRODUCTION, labels=LABELS)
+    instance_id = "snipt-inst-del" + UNIQUE_SUFFIX
+    instance = client.instance(instance_id, instance_type=PRODUCTION, labels=LABELS)
     cluster = instance.cluster(
-        "clus-my-123",
+        "clus-to-delete" + UNIQUE_SUFFIX,
```
Comment (mf2199) [inline bigtable/docs/snippets.py]: Some nit-picking, but I'd use somewhat more informative names, such as "snippet-instance", "cluster-to-delete" etc.

File: bigtable/docs/snippets.py (hunk 1 above)
Comment (sangramql) [reply bigtable/docs/snippets.py]: > Yes, not sure why this duplicate import was there before, in the first place.

These are snippets which are included in doc string of actual functions, so for example codes we need to have imports for users to understand.

**Comment by sangramql** [comment]
> The code looks fine but the questions are whether the flakes were reproduced, how positive are we about this way of solving them, and if enough testing has been done afterwards to ensure they're gone.

This issue is not consistent in production i suppose. And I was not able to reproduce even once.
I tried different approach to hit this issue,

1.    Created multiple tables(>100) in test function.
2.    Created instance and immediately created multiple tables (> 100) in class setup.
3.    Tried to created multiple instances (>10) and table in an instance, but got the following error 
after creating 6 instances. But didn’t get ‘deadline exceeded’ error though.
4. Executed parallel threads of snippets.py from nox.

File: bigtable/docs/snippets.py (hunk 2 above)
Comment (sangramql) [reply bigtable/docs/snippets.py]: This instance is created for only 'instance delete' test, hence the names.
Already tried naming that, but got string length error.

`from_value = <_Rendezvous of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMENT
	details = "Error in field 'clusters' ...lection clusters : Length should be between [6,30], but found 31 'cluster-to-delete-1564545637619'","grpc_status":3}"
  ???`
`E   google.api_core.exceptions.InvalidArgument: 400 Error in field 'clusters' : Cluster Id : Invalid id for collection clusters : Length should be between [6,30], but found 31 'cluster-to-delete-1564545637619'`

**Comment by sangramql** [comment]
Closing this PR as this opened in #8879

**Pull request closed without merging**